### PR TITLE
Call globalInitializeTracing only once

### PR DESCRIPTION
**Pull request by tudorms**
Make sure globalInitializeTracing is only called once
Also port misc changes from master

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/v8-jsi/pull/104)

### CODE REVIEW

File: scripts/build.ps1 [hunk 1]
```diff
@@ -23,34 +23,36 @@ Push-Location (Join-Path $workpath "v8build\v8")
 # Generate the build system
 $gnargs = 'v8_enable_i18n_support=false is_component_build=false v8_monolithic=true v8_use_external_startup_data=false treat_warnings_as_errors=false'
 
-if ($Configuration -like "*android") {
-    $gnargs += ' use_goma=false target_os=\"android\" target_cpu=\"' + $Platform + '\"'
+# commenting this out for now since Android build is at android/build.sh
+
+# if ($Configuration -like "*android") {
```
Comment (NikoAri) [inline scripts/build.ps1:28]: ># if  [](http://example.com/codeflow?start=0&length=5)

(nit) should all this just be removed?

File: scripts/fetch_code.ps1 [hunk 2]
```diff
@@ -15,13 +15,14 @@ $env:GIT_REDIRECT_STDERR = '2>&1'
 Push-Location (Join-Path $workpath "v8build")
 & fetch v8
 
-if ($Configuration -like "*android") {
-    $target_os = @'
-target_os= ['android']
-'@
+# commenting this out since the Android build is in android/build.sh
+# if ($Configuration -like "*android") {
```
Comment (NikoAri) [inline scripts/fetch_code.ps1:19]: (nit) same comment as in other file, Remove this?

File: src/V8JsiRuntime.cpp [hunk 3]
```diff
@@ -446,9 +446,11 @@ void V8Runtime::createHostObjectConstructorPerContext() {
       isolate_, constructorForHostObjectTemplate->GetFunction(context_.Get(isolate_)).ToLocalChecked());
 }
 
+std::once_flag g_tracingInitialized;
```
Comment (NikoAri) [inline src/V8JsiRuntime.cpp:449]: >g_tracingInitialized [](http://example.com/codeflow?start=15&length=20)

(nit) should this be declared inside of ifdef _WIN32 clause?

**Review by NikoAri — APPROVED** [review]
:shipit:

File: src/V8JsiRuntime.cpp (hunk 3 above)
Comment (tudorms) [reply src/V8JsiRuntime.cpp:450]: Thanks, good catch.

File: scripts/build.ps1 (hunk 1 above)
Comment (tudorms) [reply scripts/build.ps1:28]: I'll leave the scripts like this for now to match master.

**Pull request merged**